ethnic_analysis/ethnic_analysis.sql

```diff
@@ -0,0 +1,363 @@
+group by candidate, party
+order by party, how_many_times_winner desc;
+
+--hispanic_or_latino--
+create temp table working_version_hispanic_or_latino
+as
+select 
+	cw.state
+,	cw.state_abbreviation
+,	cw.candidate
+,	cw.party
+,	cw.summary_votes
+,	cw.hispanic_or_latino_percentage
+,	(case when "candidate" like 'Hillary Clinton' then 1
+		 when "candidate" like 'Bernie Sanders' then 2
+		 when "candidate" like 'Donald Trump' then 3
+		 when "candidate" like 'Ted Cruz' then 4
+		 when "candidate" like 'Jeb Bush' then 5
+		 when "candidate" like 'Carly Fiorina' then 6
+		 when "candidate" like 'Martin O''Malley' then 7
+		 when "candidate" like 'John Kasich' then 8
+		 when "candidate" like 'Rand Paul' then 9
+		 when "candidate" like 'Chris Christie' then 10
+		 when "candidate" like 'Ben Carson' then 11
+		 when "candidate" like 'Mike Huckabee' then 12
+		 when "candidate" like 'Marco Rubio' then 13
+		 else 0 end) as ile
+from candidate_winners cw
+where cw.hispanic_or_latino_percentage > (select avg(cf.rhi725214) from county_facts cf)
+group by cw.state, cw.state_abbreviation, cw.candidate, cw.party, cw.summary_votes, cw.hispanic_or_latino_percentage;
+
+select 
+	candidate
+,	party
+,	count("ile") as how_many_times_winner
+from working_version_hispanic_or_latino
+group by candidate, party
+order by party, how_many_times_winner desc;
+
+--two_or_more_races--
+
+create temp table working_version_two_or_more_races
+as
+select 
+	cw.state
+,	cw.state_abbreviation
+,	cw.candidate
+,	cw.party
+,	cw.summary_votes
+,	cw.two_or_more_races_percentage
+,	(case when "candidate" like 'Hillary Clinton' then 1
+		 when "candidate" like 'Bernie Sanders' then 2
+		 when "candidate" like 'Donald Trump' then 3
+		 when "candidate" like 'Ted Cruz' then 4
+		 when "candidate" like 'Jeb Bush' then 5
+		 when "candidate" like 'Carly Fiorina' then 6
+		 when "candidate" like 'Martin O''Malley' then 7
+		 when "candidate" like 'John Kasich' then 8
+		 when "candidate" like 'Rand Paul' then 9
+		 when "candidate" like 'Chris Christie' then 10
+		 when "candidate" like 'Ben Carson' then 11
+		 when "candidate" like 'Mike Huckabee' then 12
+		 when "candidate" like 'Marco Rubio' then 13
+		 else 0 end) as ile
+from candidate_winners cw
+where cw.two_or_more_races_percentage > (select avg(cf.rhi625214) from county_facts cf)
+group by cw.state, cw.state_abbreviation, cw.candidate, cw.party, cw.summary_votes, cw.two_or_more_races_percentage;
+
+select 
+	candidate
+,	party
+,	count("ile") as how_many_times_winner
+from working_version_two_or_more_races
+group by candidate, party
+order by party, how_many_times_winner desc;
+
+--sparawdzam czarnosk�rego kandydata Ben Carson w jakich stanach najepiej mu posz�o--
+--czy w tych stanach procent os�b czarnosk�rych by� wy�szy ni� przeci�tnie--
+
+select * from candidate_winners
+where "candidate" like 'Ben Carson';
+
+
+select*from summary_votes scv;
+select*from state_summary;
+
+select scv.* from summary_votes scv
```

Comment:
Jak już w całym skrypcie stosujesz konwencję rozbijania ze względu na słowa kluczowe to warto robić to konsekwentnie w każdym dłuższym zapytaniu